packages/material-ui/src/Stack/Stack.d.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { SystemProps, SxProps } from '@material-ui/system';
+import { OverrideProps, OverridableComponent } from '../OverridableComponent';
+import { Theme } from '../styles/createMuiTheme';
+
+export type StackSpacing = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
```

Comment:
I imagine you have used `GridSpacing` as inspiration.

```suggestion
```

The enum was required for JSS that has static generated values, with the styled API, we can accept a number. As for exporting the type, it useless now, developers can use `StackProps['spacing']`.